README.rst

```diff
@@ -3,19 +3,65 @@ CLaR
 
 |image0| |image1|
 
-This package implements the CLaR alorithm, a fast algorithm to handle sparse linear regression with heteroscedastic noise, see:
-Bertrand, Q., Massias, M., Gramfort, A., & Salmon, J. (2019). Handling correlated and repeated measurements with the smoothed Multivariate square-root Lasso. NeurIPS (2019). arXiv preprint arXiv:1902.02509.
+Fast algorithm to solve high dimensional linear regression with correlated noise and multiple measurements
 
-It also implements a variation of the MRCE algorithm with a L21 penalization on the regression coefficient Beta, and which take in account the repetitions see
-Rothman, A. J., Levina, E., & Zhu, J. (2010). Sparse multivariate regression with covariance estimation. Journal of Computational and Graphical Statistics, 19(4), 947-962.
 
+Documentation
+=============
 
-To be able to run the code you first need to run, in this folder:
-```pip install -e .```
+Please visit https://qb3.github.io/CLaR/ for the latest version
```

Comment:
CLaR > clar everywhere (in python and URLs) ? @QB3

Reply:
maybe this deserves a separated PR? wdyt?

Reply:
well we can always start by making stuff clean here in the readme